WordPress/Classes/ViewRelated/Me/App Settings/About/UnifiedAboutHeaderView.swift

```diff
@@ -0,0 +1,128 @@
+import Foundation
+import SwiftUI
+
+final class UnifiedAboutHeaderView: UIView {
+
+    // MARK: - Customization Support
+
+    struct AppInfo {
+        let icon: UIImage
+        let name: String
+        let version: String
+    }
+
+    struct Spacing {
+        let betweenAppIconAndAppNameLabel: CGFloat
+        let betweenAppNameLabelAndAppVersionLabel: CGFloat
+        let aboveAndBelowHeaderView: CGFloat
+    }
+
+    struct Sizing {
+        let appIconWidthAndHeight: CGFloat
+        let appIconCornerRadius: CGFloat
+    }
+
+    struct Fonts {
+        let appName: UIFont
+        let appVersion: UIFont
+    }
+
+    // MARK: - Defaults
+
+    public static let defaultSizing = Sizing(
+        appIconWidthAndHeight: CGFloat(80),
+        appIconCornerRadius: CGFloat(13))
+
+    public static let defaultSpacing = Spacing(
+        betweenAppIconAndAppNameLabel: CGFloat(16),
+        betweenAppNameLabelAndAppVersionLabel: CGFloat(4),
+        aboveAndBelowHeaderView: CGFloat(64))
+
+    // MARK: - View Customization
+
+    private let appInfo: AppInfo
+    private let spacing: Spacing
+    private let sizing: Sizing
+    private let fonts: Fonts
+
+    // MARK: - Initializers
+
+    init(appInfo: AppInfo,
+         sizing: Sizing = defaultSizing,
+         spacing: Spacing = defaultSpacing,
+         fonts: Fonts) {
+
+        self.appInfo = appInfo
+        self.sizing = sizing
+        self.spacing = spacing
+        self.fonts = fonts
+
+        super.init(frame: .zero)
+
+        setupSubviews()
+    }
+
+    override init(frame: CGRect) {
+        fatalError("Initializer not implemented!")
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("Initializer not implemented!")
+    }
+
+    // MARK: - Setting up the subviews
+
+    func setupSubviews() {
+        let stackView = UIStackView()
+        let iconView = UIImageView()
+        let appNameLabel = UILabel()
+        let appVersionLabel = UILabel()
+
+        clipsToBounds = true
+
+        appNameLabel.text = appInfo.name
+        appNameLabel.lineBreakMode = .byWordWrapping
+        appNameLabel.numberOfLines = 1
+        appNameLabel.font = fonts.appName
+
+        appVersionLabel.text = appInfo.version
+        appVersionLabel.lineBreakMode = .byWordWrapping
+        appVersionLabel.numberOfLines = 1
+        appVersionLabel.font = fonts.appVersion
+        appVersionLabel.textColor = UIColor.colorFromHex("8A898E")
```

Comment:
We might just want to use `.textSubtle` here? (or even `.secondaryLabel` to make it more standard and portable)

Reply:
Yes, this was a pending question.  The color is non-standard so I'll ping Ann about this.

Reply:
Changed to `.secondaryLabel` and pinged Ann about this.  IMHO we can go ahead and if she asks for a different color I can adjust in this PR if it's still open or a new one if not.